subprojects/wrapper/src/integTest/groovy/org/gradle/integtests/WrapperLoggingIntegrationTest.groovy

```diff
@@ -122,6 +122,6 @@ class WrapperLoggingIntegrationTest extends AbstractWrapperIntegrationSpec {
         result = wrapperExecuter.run()
 
         then:
-        outputContains("........10%.........20%.........30%.........40%.........50%.........60%.........70%.........80%.........90%.........100%")
+        result.getOutputLineThatContains("10%").replace('.', '') == '10%20%30%40%50%60%70%80%90%100%'
```

Comment:
How about replacing a sequence of dots with `|` and then assert that separator as well?

Reply:
Indeed, let me try that.